### PR TITLE
Add release quality details

### PR DESCRIPTION
**Pull request by andreasohlund**
Is this the right home for this?

If yes we'll update platdev to link here and only have specific instructions related to our internal process in platdev

### CODE REVIEW

**Comment by SimonCropp** [comment]
LGTM

**Comment by andreasohlund** [comment]
[WIP] removed. Will open a platdev PR to remove the redundant text there once this is merged

File: nservicebus/upgrades/release-policy.md [hunk 1]
```diff
@@ -41,9 +42,9 @@ Some examples:
...
+It is strongly recommend to upgrade frequently enough to stay on a supported version. For a best upgrade experience upgrade from major version to major version. For example if using NServiceBus Version 3.3.15 and want to upgrade to NServiceBus Version 5.2.3. First upgrade to the latest NServiceBus Version 4.x release (i.e. Version 4.7.6). Follow the suggested API upgrade guides and [ObsoleteAttribute messages](https://msdn.microsoft.com/en-us/library/system.obsoleteattribute.aspx) and then upgrade to NServiceBus Version 5.2.3.
 
-[Contact support](https://particular.net/support) if there are any bugfixes that believe should be back-ported to a specific. version
+[Contact support](https://particular.net/support) if there are any bugfixes that believe should be back-ported to a specific version.
```
Comment (indualagarsamy) [inline nservicebus/upgrades/release-policy.md]: >[Contact support](https://particular.net/support) if there are any bugfixes that believe should be back-ported to a specific version.

How about:
[Contact support](https://particular.net/support) if there are any bugfixes that needs to be back-ported to a specific version.

File: nservicebus/upgrades/release-policy.md [hunk 2]
```diff
@@ -79,3 +80,45 @@ Since all critical issues will be back-ported, choose to upgrade when convenient
...
+ * APIs might change without notice.
+ * Packages are used for internal development, updating docs, etc.
+
+
+### Beta
+
+ * The component is feature complete.
+ * Likely to contain a number of known or unknown bugs.
+ * Performance and stability testing has not been fully completed.
+ * **No** "go live" license is provided and production use is **not supported**.
```
Comment (indualagarsamy) [inline nservicebus/upgrades/release-policy.md]: >**No** "go live" license is provided and production use is **not supported**.

How about:

This version does not come with a "go live" license and will **not be supported** in Production.

File: nservicebus/upgrades/release-policy.md [hunk 3]
```diff
@@ -79,3 +80,45 @@ Since all critical issues will be back-ported, choose to upgrade when convenient
...
+ * All public APIs should be stable but may change based on feedback from consumers.
+
+
+#### Open and closed beta
```
Comment (indualagarsamy) [inline nservicebus/upgrades/release-policy.md]: Why does this distinction matter?
People who have access to this are public. If you're adding this, how about you also add something that states that, if someone is interested in being part of the closed group beta, they should contact us?

File: nservicebus/upgrades/release-policy.md [hunk 4]
```diff
@@ -79,3 +80,45 @@ Since all critical issues will be back-ported, choose to upgrade when convenient
...
+
+ * Closed beta versions are released to a restricted group customers by invitation.
+ * Open beta versions are public and target the entire user base.
+
+
+### Release Candidate (RC)
+
+A Release Candidate is a version intended to be a final product, which is ready to release unless significant bugs emerge. In this stage of stabilization, all features have been designed, coded and tested through one or more beta cycles and has no known critical bugs. There could still be source code changes to fix defects, changes to documentation and data files, and peripheral code for test cases or utilities.
```
Comment (indualagarsamy) [inline nservicebus/upgrades/release-policy.md]: Might also be good to talk about the go-live license and the support in production here.

File: nservicebus/upgrades/release-policy.md [hunk 5]
```diff
@@ -79,3 +80,45 @@ Since all critical issues will be back-ported, choose to upgrade when convenient
...
+
+
+### Release To Market (RTM)
+
+This is the stable production release.
```
Comment (indualagarsamy) [inline nservicebus/upgrades/release-policy.md]: Same thing with the go live license.

**Comment by indualagarsamy** [comment]
@andreasohlund @SimonCropp - I have made the changes to this PR, that I recommended. Have a look.

File: nservicebus/upgrades/release-policy.md (hunk 5 above)
Comment (SimonCropp) [inline nservicebus/upgrades/release-policy.md]: Did u miss this one?

Comment (indualagarsamy) [inline nservicebus/upgrades/release-policy.md]: @SimonCropp - After thinking about it some more, if this is the release to market, support is implied. So I didn't add it. Do you think I need to add it?

**Pull request merged**